### PR TITLE
SVG optimizations

### PR DESCRIPTION
**Pull request by jfschwarz**
This is a first batch of some optimisations aiming at making minting wands cheaper (saving storage space).

- store stone params as ints instead of strings
- add dynamic seed to stone texture (if we use the tokenId, for example, every stone would be unique. let's see if we want this. if yes, I believe it would make the rotation param obsolete)
- remove planet names from storage
- adjust forging UI

### CODE REVIEW

File: contracts/contracts/svg/partials/stone.hbs [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
+    seed="{{uint seed}}"
   ></feTurbulence>
   <feComponentTransfer>
     <feFuncR
-      id="r"
       type="gamma"
-      amplitude="{{redAmp}}"
-      exponent="{{redExp}}"
-      offset="{{redOff}}"
+      amplitude="{{int8 redAmp decimals=2}}"
```
Comment (samepant) [inline contracts/contracts/svg/partials/stone.hbs:11]: nice, i'm assuming storing ints is less space than floats?

**Review by samepant — APPROVED** [review]
Looks good!! 🥂

File: app/components/SvgTemplate.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ export interface StoneSettings {
   blueExp: number;
   blueOff: number;
   rotation: number;
+  seed: number;
```
Comment (samepant) [inline app/components/SvgTemplate.tsx:49]: It'd be great to use an account address for the seed, as that would let us show how the seed affects the stone in the minting app when a wallet is connected.

File: contracts/contracts/svg/partials/stone.hbs (hunk 1 above)
Comment (jfschwarz) [reply contracts/contracts/svg/partials/stone.hbs:11]: Solidity doesn't have floats, so ints with fixed points and are our only option

**Pull request merged**